doc/source/whatsnew/v2.1.0.rst

```diff
@@ -59,14 +59,16 @@ and ``na_action="ignore"`` did not work correctly for any ``ExtensionArray`` sub
     ser = pd.Series(["a", "b", np.nan], dtype="category")
     ser.map(str.upper, na_action="ignore")
     df = pd.DataFrame(ser)
-    df.applymap(str.upper, na_action="ignore")
+    df.map(str.upper, na_action="ignore")
     idx = pd.Index(ser)
     idx.map(str.upper, na_action="ignore")
 
 Also, note that :meth:`Categorical.map` implicitly has had its ``na_action`` set to ``"ignore"`` by default.
 This has been deprecated and will :meth:`Categorical.map` in the future change the default
 to ``na_action=None``, like for all the other array types.
 
+Notice also that in this version, :meth:`DataFrame.applymap` has been renamed to :meth:`DataFrame.map` (:issue:`52353`).
```

Comment:
I think a possible reading of this, and L28 above, is that applymap no longer exists. Can we say something more like "map has been added and applymap is deprecated, users should switch to map"